tests/roles/development_environment/tasks/main.yaml

```diff
@@ -46,3 +46,18 @@
 - name: debug token
   ansible.builtin.debug:
     var: before_adoption_token
+
+- name: create credential for sanity checking its value after adoption
+  no_log: "{{ use_no_log }}"
+  environment:
+    INSTALL_YAMLS_PATH: "{{ install_yamls_path }}"
```

Comment:
do we need this variable? I might be missing something, but this seems to be executing only and `openstack` command

Reply:
+1 but looks like the snippets above suffer from the same problem so this can be cleaned up separately i think.